README.md

```diff
@@ -87,6 +87,38 @@ The simulate method initializes a simulation in the cloud. In this call, we set
 
 When the simulation ends, the simulation output files can be found in the output_dir. Also, the object simulation can be used to visualize some aspects of the outputs. In particular, ```output.render_interactive()``` yields an interactive visualization that can be visualized in a standard jupyter notebook.
 
+### ProteinSolvation
+
+This scenario simulates the dynamics of a protein whose structure is described by a PDB file. The protein is placed in a cubic box filled with water. If the protein has a non-zero electric charge, charged ions are added to the solution to neutralize the system. The position of the atoms in this system is updated using Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields.
+
+#### Example
+
+First we initialize the scenario:
+```
+from inductiva.molecules.scenarios.protein_solvation import ProteinSolvation
```

Comment:
can we simplify the import? 

from inductiva.molecules.scenarios import ProteinSolvation